lib/saddle/middleware/request/retry.rb

```diff
@@ -40,7 +40,11 @@ def call(env)
         end
 
         def self.deep_copy(value)
-          if value.is_a?(Hash)
+          if value.is_a?(Struct)
+            result = value.clone
+            value.each{|k, v| result[k] = deep_copy(v)}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.